responsibleai/responsibleai/_interfaces.py

```diff
@@ -94,7 +94,8 @@ class CausalPolicyTreeLeaf:
 class CausalPolicyTreeInternal:
     leaf: False
     feature: str
-    threshold: Union[float, str]  # TODO: Categorical features
+    category: Optional[str]
```

Comment:
so category and threshold are exclusive here, correct?

I was think of adding a sort of comparation type field and keep threshold as is.
For current regression, it will be type = "SmallerThan" threshold = 123
Categorical feature: type = "Equal" threshold="class 1"

So that we can expand in future to support other type, eg: in multiple category,

Reply:
@xuke444 I've addressed your comment. Please help me consider the back-compat ramifications that changing this might have on the UI.

Reply:
There should not be any issue, ui will still shows same thing as before, we will need to make code change to utilize new field.

Reply:
Great, thanks!